eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -75,23 +76,42 @@ public RegionWSServerV2() {
     @GetMapping(value = "/{regionId}/variants")
     @ResponseBody
     @RateLimit(value = REGION_REQUEST_RATE_LIMIT)
-    public ResponseEntity getVariantsByRegion(@PathVariable("regionId") String regionId,
-                                              @RequestParam(name = "species") String species,
-                                              @RequestParam(name = "assembly") String assembly,
-                                              @RequestParam(name = "studies", required = false) List<String> studies,
-                                              @RequestParam(name = "annot-ct", required = false) List<String>
-                                                      consequenceType,
-                                              @RequestParam(name = "maf", required = false) String maf,
-                                              @RequestParam(name = "polyphen", required = false) String polyphenScore,
-                                              @RequestParam(name = "sift", required = false) String siftScore,
-                                              @RequestParam(name = "annot-vep-version", required = false) String
-                                                      annotationVepVersion,
-                                              @RequestParam(name = "annot-vep-cache-version", required = false) String
-                                                      annotationVepCacheVersion,
-                                              @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                              @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                              HttpServletResponse response,
-                                              @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByRegion(
+            @ApiParam(value = "Comma separated values of regions, in format chr:start-end. e.g. 13:32884600-32884700")
```

Comment:
Move the sample region to an actual example, for illustration purposes of a query.
```suggestion
            @ApiParam(value = "Comma separated genomic regions in the format chr:start-end.")
```